src/lib/storage/zero_suppression/encode_by_zs_type.cpp

```diff
@@ -0,0 +1,44 @@
+#include "encode_by_zs_type.hpp"
+
+#include <map>
+#include <memory>
+
+#include "utils/assert.hpp"
+
+// Include your zero suppression encoder file here!
```

Comment:
This comment seems redundant. When people add their encoder in the list below, they will soon see that they also have to include their header...